scripts/world_bank/wdi/worldbank.py

```diff
@@ -85,8 +229,8 @@ def read_worldbank(iso3166alpha3, fetchFromSource):
             Takes approximately 10 seconds to download and
             tidy one country in a Jupyter notebook.
     """
-    if fetchFromSource:
-        logging.info('Downloading %s', iso3166alpha3)
+    if mode in ["download", '']:
+        logging.info('Downloading input file for country %s', iso3166alpha3)
         country_zip = ("http://api.worldbank.org/v2/en/country/" +
```

Comment:
Please ensure that the downloaded file is saved in its original format to a folder. The current code appears to only save the processed file.

Reply:
Modified to save download file in its original format to a folder